### DISCUSSION TITLE
Welcome to Crocrodile Discussions!

### DISCUSSION THREAD

**Discussion by Virinas-code** [Virinas-code/Crocrodile#34] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vibranium21** [comment]
I’m Vibranium21, I occasionally check out the code, i found crocodile in May when I was interested in lichess bots
I am currently interested in data-viz (data graphing) but i still check in with the repo every few days.

**Reply by Virinas-code** [reply]
Thank you! Check out white-rabbit too, his little brother [View on GitHub :arrow_right:](https://github.com/Virinas-code/white-rabbit).
